packages/blade/src/components/Dropdown/FilterChipSelectInput.web.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unnecessary-type-assertion */
+
+import React from 'react';
+import { useDropdown } from './useDropdown';
+import { dropdownComponentIds } from './dropdownComponentIds';
+import { assignWithoutSideEffects } from '~utils/assignWithoutSideEffects';
+import { BaseFilterChip } from '~components/FilterChip/BaseFilterChip';
+import { getActionListContainerRole } from '~components/ActionList/getA11yRoles';
+import type { BaseFilterChipProps } from '~components/FilterChip/types';
+import type { DataAnalyticsAttribute } from '~utils/types';
+
+type FilterChipSelectInput = Pick<
```

Comment:
```suggestion
type FilterChipSelectInputProps = Pick<
```

seems to clash with component name